src/pages/LoginPage/LoginPage.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
+
+import { FormWrapper } from 'components/LoginLayout/FormWrapper'
+import { Button } from 'components/Button/Button'
+import { Input } from 'components/Input/Input'
+
+import styles from './LoginPage.module.scss'
+
+export const LoginPage = () => {
+  const [form, setValues] = useState({
+    username: '',
+    password: '',
+  })
+
+  const handleChange = fildName => value => {
+    setValues({
+      ...form,
+      [fildName]: value,
+    })
+  }
+
+  const handleSubmit = () => {}
+
+  return (
+    <FormWrapper>
+      <div className={styles.form}>
```

Comment:
Why you added `<div className={styles.form}>` here? Why not inside of `FormWrapper` ?